homeworks/vazgen_harutyunyan/Homework_7/Numbers.java

```diff
@@ -0,0 +1,21 @@
+public class Numbers {
+    public static void main(String[] args) {
+        digitToWord(1230);
+
+
+    }
+
+
+
+
+    static String digitToWord(int number) {
+        String[] array = {"Zero", "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine"};
+        if(number <= 9) {
+            System.out.println(array[number]);
+            return array[number];
+        } else {
+            System.out.println(array[number%10]);
+            return digitToWord(number / 10);
+        }
+    }
```

Comment:
you need to convert 6843 int to 6843 string with recursion and keeping the order.
The array is not needed and the order isn't kept